src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -50,20 +50,24 @@
  * </li>
  * </ul>
  * <p>
- * To prevent {@code FooCheck} violations from being reported write:
- * </p>
- * <pre>
- * &#64;SuppressWarnings("foo") interface I { }
- * &#64;SuppressWarnings("foo") enum E { }
- * &#64;SuppressWarnings("foo") InputSuppressWarningsFilter() { }
- * </pre>
- * <p>
  * Some real check examples:
  * </p>
  * <p>
  * This will prevent from invocation of the MemberNameCheck:
  * </p>
  * <pre>
+ * &lt;module name=&quot;TreeWalker&quot;&gt;
+ * &lt;module name=&quot;MemberName&quot;/&gt;
```

Comment:
All sub modules under TreeWalker should have +2 in indentation
To show that they are under TreeWalker.

Reply:
done